device/samples/javascript/device_methods.js

```diff
@@ -3,10 +3,15 @@
 
 'use strict';
 
-require('es5-shim');
+// Choose a protocol by uncommenting one of these transports.
 var Protocol = require('azure-iot-device-mqtt').Mqtt;
```

Comment:
nit: `var` is used for the first one and `const` is used for the others. We should use `var` or `const` but not both.
`let`/`const` is preferable but a lot of the sample code uses `var` so changing all of the `var`s might not be in scope for this PR.

Reply:
I agree. But that would require more clean up and testing. lets make it `var` for now and I will go ahead and do a separate PR later using `const` / `let` instead of `var`.